java/ql/src/semmle/code/java/dataflow/FlowSources.qll

```diff
@@ -301,6 +302,22 @@ class EnvReadMethod extends Method {
   }
 }
 
+private class PortletRenderRequestGetParameterMethod extends Method {
+  PortletRenderRequestGetParameterMethod() {
+    exists(RefType c, Interface t |
+      c.extendsOrImplements*(t) and
+      t.hasQualifiedName("javax.portlet", "RenderState") and
+      this = c.getAMethod()
+    |
+      this.hasName([
+          "getCookies", "getParameter", "getRenderParameters", "getParameterNames",
+          "getParameterValues", "getParameterMap"
+        ])
+      // TODO consider getRemoteUser
```

Comment:
Address comment or expand to note why it is not done yet

Reply:
removed it now. A taint obtained from that field is unlikely to be actually exploitable.

Reply:
Change does not appear to be pushed?